EIC-EA/EIC-community-D8
# EICNET-1126: Changed copy on about page.

Tests:

- Create a visibility public and joining methods open group
- Create a visibility public and joining methods request group
- Create a visibility private group
- Create a visibility community members only and joining methods open group
- Create a visibility community members only and joining methods request group
- *Then you should see the correct copy*
- Create a visibility custom restrictions and joining methods open group
- Create a visibility custom restriction and joining methods request group
- *Then you should see the correct copy and the custom restrictions*

Bonus bugfix:

- Wiki group header is not displayed twice any more
